pkgs/development/python-modules/canonicaljson/default.nix

```diff
@@ -4,11 +4,11 @@
 
 buildPythonPackage rec {
   pname = "canonicaljson";
-  version = "1.1.4";
+  version = "1.3.0";
```

Comment:
```suggestion
  version = "1.3.0";
  disabled = isPy27;
```
```
  Processing ./canonicaljson-1.3.0-py2-none-any.whl
  ERROR: Package 'canonicaljson' requires a different Python: 2.7.18 not in '~=3.5'
```

Reply:
otherwise LGTM
```
https://github.com/NixOS/nixpkgs/pull/95708
2 packages failed to build:
python27Packages.canonicaljson python27Packages.signedjson

7 packages built:
element-desktop matrix-synapse matrixcli python37Packages.canonicaljson python37Packages.signedjson python38Packages.canonicaljson python38Packages.signedjson
```

Reply:
Fixed.